src/index.html

```diff
@@ -11,6 +11,44 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form class="form"
+      action="/search"
+      method="get"
+      data-qa="big"
+    >
+      <img
+      class="search-icon"
+      src="images/Search.svg"
+      alt="Search-bar"
+      >
```

Comment:
```suggestion
      <img
        class="search-icon"
        src="images/Search.svg"
        alt="Search-bar"
      >
```
Fix all cases. Also, it would be great to implement the logo using background-image property
![image](https://github.com/mate-academy/layout_search-bar-airbnb/assets/96409563/33329e2b-fa5a-4eca-b06f-3e40ed4a5f55)